### PR TITLE
[RAPPS-DB] Add Mario & Luigi game (mario-luigi) 3.0.0

### PR DESCRIPTION
**Pull request by marjuanm**
Mario Bros fangame that works under MS-DOS, in versions of Windows 9x it works without problems, in Windows XP I haven't had the opportunity to try it, but in more recent versions of Windows it only works from DosBox, but in Reactos it works perfectly.

### CODE REVIEW

**Comment by marjuanm** [comment]
DOS game running on Reactos directly:
Screenshot_1
Screenshot_2
Screenshot_3
Screenshot_4

**Comment by learn-more** [comment]
@marjuanm you have both games in this PR now.

Please create a branch per pull request, because new pushes to a branch with an open pull request end up in that PR.

**Comment by JoachimHenze** [comment]
yes, agwak shouldn't be in here.

**Comment by marjuanm** [comment]
deleted incorrect files in this game

**Review by learn-more — APPROVED** [review]

**Review by katahiromz — CHANGES_REQUESTED** [review]
One request for one pull request.

File: icons/icon.ico
Comment (katahiromz) [inline icons/icon.ico:1]: What's this icon?

**Review by katahiromz — CHANGES_REQUESTED** [review]

**Review by katahiromz — APPROVED** [review]
Tests successful. LGTM.

**Pull request merged**